tapdb/universe_test.go

```diff
@@ -460,6 +472,20 @@ func TestUniverseLeafQuery(t *testing.T) {
 		targetKey.MintingOutpoint = rootMintingPoint
 
 		leaf := randMintingLeaf(t, assetGen, id.GroupKey)
+		if id.GroupKey != nil {
+			// All assets are sharing the same genesis and group
+			// key, so they must also shae the same group witness.
```

Comment:
shae -> share

Reply:
Is this for the case of a multi-mint, so same asset just with diff UTXOs? If not, then totally possilbe (if they're reissuance), that these values are the same, but they have a unique witness.

Reply:
It shouldn't be possible in practice because geneses can never be the same, even in the same batch (asset tags must be unique per-batch).

I think the closest real-world case would be a multi-asset group in the same batch, but for this test we randomly choose to use a group key for the universe ID, so I decided to match that randomization vs. forcing a group key.